Makefile

```diff
@@ -22,6 +24,20 @@ help:
 	@echo ""
 	@echo "Running 'make' without a target is equivalent to running 'make build run'."
 
+## dev: Run the excretor in development mode
+dev:
+	@echo "Starting tummy with exposed port"
+	@ZIPFILE_PATH=. $(DOCKER_COMPOSE) up tummy-dev -d --wait
+	@echo ""
+	@echo "Starting excretor in development mode. Please run 'make dev-stop' to stop the tummy later."
+	@$(EXCRETOR_DEV_ENVS) cargo run --manifest-path excretor/Cargo.toml
+
+## dev-stop: Stop the tummy-dev docker container
+dev-stop:
```

Comment:
Can add the `dev-stop` commands after the `cargo run` command in `dev` to stop the dev tummy once the excretor server stops (eg by ctrl - C)